deployment/deployment_utils.sh

```diff
@@ -34,27 +34,19 @@ build_directories () {
   echo "Assembling deployment folder."
   mkdir -p "$target_dir"
 
-  mkdir -p $target_dir/jar;
-  mkdir -p $target_dir/dataland-keycloak/dataland_theme/login;
-  mkdir -p $target_dir/dataland-frontend/src/assets
   mkdir -p $target_dir/dataland-keycloak/users;
 
-  envsubst < environments/.env.template > "$target_dir"/.env
+  cat ./*github_env.log > "$target_dir"/.env
+  set -o allexport
+  source "$target_dir"/.env
+  set +o allexport
```

Comment:
Der Sinn erschließt sich mir nicht. Hier gibt es doch keine sub-shells

Reply:
Das folgende `envsubst` läuft in einer subshell.
Wir wollen die docker-image-versions im envsubst schon vorhanden haben.

Reply:
![image](https://user-images.githubusercontent.com/47553592/200833974-e9ad9b5d-1c75-4aa1-96c9-fc92ffdefc80.png)
Test dass envsubst wirklch in einer subshell läuft

Reply:
ja und zudem ist das ja das utils script was ja auch sonstwo aufgerufen werden kann und dort entsprechende sub-shells haben könnte